packages/logic-utils/src/utils/documentation.ts

```diff
@@ -0,0 +1,191 @@
+import {
+  UI5DeprecatedInfo,
+  UI5SemanticModel,
+} from "@ui5-language-assistant/semantic-model-types";
+import { ReplaceFunction, forEach, unescape } from "lodash";
+
+const MAX_SNIPPET_LENGTH = 500;
```

Comment:
is a 400+ characters sentence readable in the problems view? should this limit be shorter?

Reply:
I couldn't find an example this long, do you have something long I can check it on?

Reply:
I don't have a long example, but max line length is often 80 or 120.
You can **hardcode** return a long sentence and see if it is rendered reasonably or not.